fbw-common/src/wasm/systems/systems/src/shared/mod.rs

```diff
@@ -102,6 +103,10 @@ pub trait LgciuWeightOnWheels {
 pub trait LgciuGearExtension {
     fn all_down_and_locked(&self) -> bool;
     fn all_up_and_locked(&self) -> bool;
+    fn main_down_and_locked(&self) -> bool;
+    fn main_up_and_locked(&self) -> bool;
+    fn nose_down_and_locked(&self) -> bool;
+    fn nose_up_and_locked(&self) -> bool;
```

Comment:
Are these real discrete outputs from the LGCIU?

Reply:
I saw at least that the EGPWC directly consumes this signal, so I assumed yes.
No other system is in between of LGCIU and EGPWC